src/data/finset/pointwise.lean

```diff
@@ -348,13 +348,22 @@ def singleton_monoid_hom : α →* finset α := { ..singleton_mul_hom, ..singlet
   (singleton_monoid_hom : α → finset α) = singleton := rfl
 @[simp, to_additive] lemma singleton_monoid_hom_apply (a : α) : singleton_monoid_hom a = {a} := rfl
 
+/-- The singleton operation as a `monoid_hom`. -/
+@[to_additive "The singleton operation as an `add_monoid_hom`."]
+def coe_monoid_hom : finset α →* set α :=
+{ to_fun := coe,
+  map_one' := coe_one,
+  map_mul' := coe_mul }
+
+@[simp, to_additive] lemma coe_coe_monoid_hom : (coe_monoid_hom : finset α → set α) = coe := rfl
+@[simp, to_additive] lemma coe_monoid_hom_apply (s : finset α) : coe_monoid_hom s = s := rfl
```

Comment:
Why do you need `simp` on the second lemma?

Reply:
I think the answer is "it's harmless and (very) occasionally convenient"